src/plugins/LADTable/components/LADRow.vue

```diff
@@ -109,56 +101,61 @@ export default {
     },
     methods: {
         updateValues(datum) {
-            let newTimestamp = datum[this.timestampKey];
+            let newTimestamp = this.formats[this.timestampKey].parse(datum),
+                update = false, limit;
```

Comment:
Let's define `limit` on a new line, it might be difficult to find otherwise.

Reply:
Since `update` is a boolean, let's use a boolean-appropriate prefix for it. eg. `shouldUpdate`.

Reply:
Done!